src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -44,37 +44,44 @@
 
         <v-col
           cols="12"
-          sm="7"
+          sm="9"
+          class="d-flex justify-space-between"
         >
           <v-text-field
-            class="incorporation-number"
+            class="expro-registration-number"
             filled
             hide-details
-            label="B.C. Extraprovincial Incorporation Number"
+            label="B.C. Extraprovincial Registration Number"
             readonly
+            xclearable
             :value="business.bcIdentifier"
+            @click:append="reset()"
           />
-        </v-col>
-
-        <v-col
-          cols="12"
-          sm="2"
-        >
-          <v-btn
-            id="undo-button"
-            class="float-sm-right float-none"
-            text
-            color="primary"
-            @click="reset()"
+          <v-tooltip
+            top
+            content-class="top-tooltip"
+            transition="fade-transition"
           >
-            <v-icon small>
-              mdi-undo
-            </v-icon>
-            <span>Undo</span>
-          </v-btn>
+            <template #activator="{ on }">
+              <!-- align clear button on top of incorporation number text field -->
+              <v-btn
+                class="clear-info-btn float-right mt-3 ml-n10 mr-1"
```

Comment:
`ml-n10` 😱 It makes sense though to keep things close.

How does this look on a smaller screen Sev? From my experience, this might mess things up.

Reply:
*Fortunately*, on a narrower screen, this column becomes full-width, so it's still fine.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/43824992-8ea6-416d-891f-b1d66f59daa8)

Reply:
Awesome, thanks! Resolving.